extensions/rich_text_components/Svgdiagram/protractor.js

```diff
@@ -27,7 +27,7 @@ const SVGTAGS = {
   rectangle: (
     '<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/' +
     '1999/xlink" version="1.1" width="494" height="338" viewBox="0 0 494 338' +
-    '"><desc>Created with Fabric.js 4.3.0</desc><defs></defs><rect x="0" y="' +
+    '"><desc>Created with Fabric.js 4.3.1</desc><defs></defs><rect x="0" y="' +
```

Comment:
Just a thought, should we consider putting the version part in a constant in this file and use template literal here and below? Seems like we have to keep updating this one.

Reply:
Hmm, I guess we can do that, but we will still have to update the constant. I will do that next time I need to do this change.